lgc/elfLinker/ElfLinker.cpp

```diff
@@ -154,18 +158,24 @@ class ElfLinkerImpl final : public ElfLinker {
   ArrayRef<OutputSection> getOutputSections() { return m_outputSections; }
   StringRef getStrings() { return m_strings; }
   SmallVectorImpl<ELF::Elf64_Sym> &getSymbols() { return m_symbols; }
+  SmallVectorImpl<ELF::Elf64_Rel> &getRelocations() { return m_relocations; }
   void setStringTableIndex(unsigned index) { m_ehdr.e_shstrndx = index; }
   StringRef getNotes() { return m_notes; }
 
   // Get string index in output ELF, adding to string table if necessary
   unsigned getStringIndex(StringRef string);
 
+  // =====================================================================================================================
+  // Get string index in output ELF.  Returns 0 if not found.
+  unsigned findStringIndex(StringRef string);
+
   // Find symbol in output ELF
```

Comment:
While you're at it: 
```suggestion
  // Find symbol in output ELF. Returns 0 if not found.
```
?

Reply:
done.